docs/arch/adr-15.md

```diff
@@ -0,0 +1,40 @@
+# ADR 15: Drawing tools API
+
+June 19, 2019
+
+## Context
+
+The way the drawing tools currently function on Panoptes-Front-End (PFE) have numerous issues including:
+
+- Re-rendering the DOM more than necessary
+- Updating the classification annotation on each touch or click event
```

Comment:
```suggestion
- Updating the classification annotation on each touch or pointer event
```

Reply:
I think this is what's causing the re-renders, so these two are probably the same problem stated two different ways.

Reply:
Yes, it is. I can combine this into one statement.